### DISCUSSION TITLE
How to render dynamic content

### DISCUSSION THREAD

**Discussion by ilSavo** [SAP/fundamental-ngx#12016] [Q&A]
I'm building a FAQs detail page, where the content is loaded from a database and converted into HTML with ngx-markdown. What I'm not able to achive, now, is to tell Fundamentals to also apply to the new content just added. Is it possible?

**Comment by mikerodonnell89** [comment]
What do you mean "tell Fundamentals to also apply to the new content just added"? As in, you'd like to use some fundamentals components/directives with some ngx-markdown content?
